bitnami/spring-cloud-dataflow/Chart.yaml

```
@@ -39,4 +39,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-spring-cloud-dataflow
   - https://github.com/bitnami/bitnami-docker-spring-cloud-skipper
   - https://dataflow.spring.io/
-version: 5.0.5
+version: 5.0.6
```

Please bump the minor version instead since you're including a new parameter:
```suggestion
version: 5.1.0
```